mkposters/mkposter.py

```diff
@@ -34,12 +34,12 @@ def mkposter(datadir):
 
         with md_file.open() as f:
             contents = f.read()
-        banner, left_body, right_body = contents.split("--split--")
+        bodies = contents.split("--split--")
 
-        banner = md_to_html(banner)
-        left_body = md_to_html(left_body)
-        right_body = md_to_html(right_body)
-        html_out = rf"""<!doctype html>
+        bodies = [md_to_html(b) for b in bodies]
+        banner = bodies[0]
+
+        html_out = "\n".join([f"""<!doctype html>
```

Comment:
I think this should be `rf"""....` rather than just `f"""...`? (C.f. the original code.)